src/librustc_back/target/mod.rs

```diff
@@ -71,10 +71,9 @@ macro_rules! supported_targets {
 
         // this would use a match if stringify! were allowed in pattern position
```

Comment:
> (it wasn't true anyway)

Actually it would look like this: http://is.gd/ot9Ta2

Now that we are using `$triple` instead of `$module` we can change this into a match. I'll send a follow up PR because this one already got pulled into the rollup.